packages/amplify-graphql-auth-transformer/src/__tests__/acm.test.ts

```diff
@@ -0,0 +1,41 @@
+import { IndexTransformer, PrimaryKeyTransformer } from '@aws-amplify/graphql-index-transformer';
+import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
+import { GraphQLTransform } from '@aws-amplify/graphql-transformer-core';
+import { AuthTransformer } from '..';
+import { ACMTest, acmTests } from './acm-test-database';
```

Comment:
Micro-nit: "database" doesn't seem like the right word here. Maybe something like "library" - but feel free to ignore me 😄